src/Sentry/SentryOptions.cs

```diff
@@ -341,7 +341,15 @@ public IDiagnosticLogger DiagnosticLogger
             get => Debug ? _diagnosticLogger : null;
             set
             {
-                _diagnosticLogger?.LogInfo("Replacing current logger with: '{0}'.", value?.GetType().Name);
+                if (value is null)
+                {
+                    _diagnosticLogger?.LogDebug("Sentry will not emit SDK debug messages because debug mode has been turned off off.");
```

Comment:
```suggestion
                    _diagnosticLogger?.LogDebug("Sentry will not emit SDK debug messages because debug mode has been turned off.");
```